docs/Developers/Development_Setup.rst

```diff
@@ -43,14 +43,22 @@ Using virtualenv
 - Install virtualenv using pip :
     ::
 
-        $ pip install virtualenv
+        $ pip3 install virtualenv
 
 - Create the virtualenv :
     ::
 
         $ cd working_dir # move into the dir where you want to create coala-venv
         $ virtualenv coala-venv
 
+NOTE:
+If you have both Python 3 and Python 2 installed try this command,
+as this creates an isolated Python 3 environment called coala-venv,
```

Comment:
Please delete `as` and start the sentence on `this`. No comma at end of line please